cxbox-core/src/main/java/org/cxbox/core/dto/rowmeta/RowDependentFieldsMeta.java

```diff
@@ -86,6 +86,11 @@ public final void setDictionaryTypeWithConcreteValuesFromList(DtoField<? super T
 		setDictionaryTypeWithConcreteValues(field, type, keys);
 	}
 
+	/**
+	 * It is not recommended to load images individually for specific values.
+	 * Instead, load all images at once as a complete reference book
+	 * use {@link FieldsMeta#setAllValuesAddIconsLOV}
+	 */
 	@Deprecated
 	public void setDictionaryValuesWithIcons(String field, IDictionaryType type, Map<LOV, IconCode> valueIconMap) {
```

Comment:
@Deprecated method is ready to be deleted

Reply:
done